File: 9-palindrome-number/palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int n,y=x,s=0;
        while(x>0){
           n=x%10;
           x=x/10;
           s=s*10+n;
        }
        if(s==y)
           return(true);
        else
           return(false);
    }
}

File: 9-palindrome-number/palindrome-number.c
bool isPalindrome(int x) {
    long digit=0,revnum=0,num=x;
    while(x>0){
        digit=x%10;
        revnum =revnum*10 + digit;
        x /=10;
    }
    return (num==revnum)?true:false;
}